components/infobox/wikis/trackmania/infobox_league_custom.lua

```diff
@@ -140,10 +150,50 @@ function CustomLeague:addToLpdb(lpdbData, args)
 
 	lpdbData.maps = table.concat(self:getAllArgsForBase(args, 'map'), ';')
 
+	lpdbData.extradata.circuit = args.circuit
+	lpdbData.extradata.circuittier = args.circuittier
+
 	-- Legacy, can be superseeded by lpdbData.mode
 	lpdbData.extradata.individual = self.data.mode == DEFAULT_MODE
 
 	return lpdbData
 end
 
+---@param widgets Widget[]
+---@param circuitIndex string|number?
```

Comment:
if this is optional you will get errors at lines 168-172

Reply:
Won't line 165 prevent this? I am setting it to null if its omitted from the call

Reply:
ah, right, didn't see it on phone